src/logic/operate.js

```diff
@@ -0,0 +1,22 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const num1 = Big(numberOne);
+  const num2 = Big(numberTwo);
+  let total = 0;
+
+  if (operation === '-') {
+    total = num1.minus(num2);
+  } else if (operation === '+') {
+    total = num1.plus(num2);
+  } else if (operation === 'X') {
+    total = num1.times(num2);
+  } else if (operation === '÷') {
+    total = num1.div(num2);
```

Comment:
Division by zero is not handled properly yet. Let's fix this.
The reason is that when a division by zero happens, we should display an error message like `zero division error`